Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
I finally implemented the numbers and indeed see a clash between the extended barrel calorimeters and the "FwdHelp": The calorimeters themselves go down to eta 2.5 but then there is 10 cm of cryo foreseen below that. The lower radius at z = 7.5 m of the cryo system corresponds to 1.223 m.